examples/connect/package.json

```diff
@@ -40,9 +40,9 @@
     "@opentelemetry/exporter-jaeger": "^0.25.0",
     "@opentelemetry/exporter-zipkin": "^0.25.0",
     "@opentelemetry/exporter-collector": "^0.25.0",
-    "@opentelemetry/instrumentation": "^0.25.0",
+    "@opentelemetry/instrumentation": "^0.34.0",
     "@opentelemetry/instrumentation-connect": "^0.24.0",
-    "@opentelemetry/instrumentation-http": "^0.25.0",
+    "@opentelemetry/instrumentation-http": "^0.34.0",
```

Comment:
The examples are terrible outdated and upgrading parts of it will likely not work. I recommend to leave them alone until someone finds time to port them to some place where at least a basic verification is done.